components/layout/footer/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import {
+  DiscordSVG,
+  GithubSVG,
+  HeartSVG,
+  IPXSVG,
+  TelegramSVG,
+  XSVG,
+} from '@/components/svg';
+
+const Footer: FC = () => (
+  <Box
+    fontSize="s"
+    display="flex"
+    fontFamily="Proto"
+    flexDirection="column"
+    alignItems="center"
+    textTransform="uppercase"
+    padding="4rem 0.75rem 1.5rem 0.75rem"
+  >
+    <Box
+      textAlign="center"
+      mb="l"
+      maxWidth="96%"
+      minWidth="90rem"
+      fontSize="inherit"
+      fontFamily="inherit"
+      justifyContent="center"
```

Comment:
This justifyContent doesn't have any effect cuz you're not using display.
U can remove it.